backend/main.py

```
@@ -1381,93 +1389,83 @@ def check_whisper_loaded(model_size: str):
-            
-            # Method 3 removed - checking for config.json is too lenient
-            # Methods 1 and 2 properly verify that model weight files exist
+                                size_mb = total_size / (1024 * 1024)
+                            except Exception as exc:
+                                processLogger.debug("Size calc (fallback) failed for %s: %s", hf_repo_id, exc)
+            except Exception as exc:
+                processLogger.debug("Fallback cache check failed for %s: %s", hf_repo_id, exc)
+
+        return downloaded, size_mb
+    
+    for config in model_configs:
```

Ruff (0.15.10)

[error] Import `config` from line 44 shadowed by loop variable

(F402)